src/CoreBundle/Field/FieldSidebarConnected.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\Field;
+
+use ChameleonSystem\CoreBundle\DataAccess\MenuItemDataAccessInterface;
+use ChameleonSystem\CoreBundle\DataModel\MenuCategoryAndItem;
+use ChameleonSystem\CoreBundle\ServiceLocator;
+
+class FieldSidebarConnected extends \TCMSFieldVarchar
+{
+    /**
+     * @var MenuItemDataAccessInterface
+     */
+    private $menuItemDataAccess;
+
+    public function __construct(?MenuItemDataAccessInterface $menuItemDataAccess = null)
+    {
+        // TODO isVirtual?
+
+        if (null === $menuItemDataAccess) {
+            $menuItemDataAccess = ServiceLocator::get('chameleon_system_core.sidebar.menu_access');
+        }
+        $this->menuItemDataAccess = $menuItemDataAccess;
+    }
+
```

Comment:
Overwrite the GetHTML() method and call the GetReadOnly() method, too, so this feld type is always read only.

Reply:
Is done by adding this method as suggested by you.